src/ocrd_anybaseocr/ocrd-tool.json

```diff
@@ -0,0 +1,136 @@
+        },
+        "rulerAreaMax": {
+          "type": "number",
+          "format": "float",
+          "default": 0.3,
+          "description": "ruler detection and suppression: maximum area of bbox (as ratio of total image pixels)"
+        },
+        "rulerAreaMin": {
+          "type": "number",
+          "format": "float",
+          "default": 0.01,
+          "description": "ruler detection and suppression: minimum area of bbox (as ratio of total image pixels)"
+        },
+        "rulerWidthMax": {
+          "type": "number",
+          "format": "float",
+          "default": 0.95,
+          "description": "ruler detection and suppression: maximum width of bbox (as ratio of total image width)"
+        },
+        "columnAreaMin": {
+          "type": "number",
+          "format": "float",
+          "default": 0.05,
+          "description": "text block detection: minimum area of individual columns (as ratio of total image pixels)"
+        },
+        "columnSepWidthMax": {
+          "type": "number",
+          "format": "float",
+          "default": 0.04,
+          "description": "text block detection: maximum width between individual columns (as ratio of total image width)"
+        },
+        "marginTop": {
+          "type": "number",
+          "format": "float",
+          "default": 0.25,
+          "description": "ruler / edge / text detection: maximum y position to crop from above (as ratio of total image height)"
+        },
+        "marginBottom": {
+          "type": "number",
+          "format": "float",
+          "default": 0.75,
+          "description": "ruler / edge / text detection: minimum y position to crop from below (as ratio of total image height)"
+        },
+        "marginLeft": {
+          "type": "number",
+          "format": "float",
+          "default": 0.3,
+          "description": "ruler / edge / text detection: maximum x position to crop from left (as ratio of total image width)"
+        },
+        "marginRight": {
+          "type": "number",
+          "format": "float",
+          "default": 0.7,
+          "description": "ruler / edge / text detection: minimum x position to crop from right (as ratio of total image width)"
+        },
+        "padding": {
+          "type": "number",
+          "format": "integer",
+          "default": 10,
+          "description": "extend / shrink border resulting from edge detection / text detection by this many px in each direction"
+        }
+      }
+    },
+    "ocrd-anybaseocr-layout-analysis": {
+      "executable": "ocrd-anybaseocr-layout-analysis",
+      "categories": [
+        "Layout analysis"
+      ],
+      "steps": [
+        "layout/analysis"
+      ],
+      "description": "Generates a table-of-content like document structure of the whole document.",
+      "input_file_grp_cardinality": 1,
+      "output_file_grp_cardinality": 0,
+      "resource_locations": ["module"],
```

Comment:
Why this restriction?

IIUC just because we currently have no other models does not mean we could not (in theory) add them, as a user.

```suggestion
```